Makefile

```diff
@@ -1,6 +1,8 @@
 TOPDIR := $(PWD)
-PLATFORMS := $(foreach platform,$(wildcard platforms/*),$(platform)/gpt)
-PARTITIONS_XML := $(foreach platform,$(wildcard platforms/*),$(platform)/partitions.xml)
+PARTITIONS := $(wildcard platforms/*/partitions.conf) $(wildcard platforms/*/*/partitions.conf)
```

Comment:
Probably good to call sort here to have a stable list as well.

Reply:
The out of order would only happen if we had a machine with platforms/<machine>/partitions.conf and also additional emmc/ufs files, and while OK from the syntax perspective, I don't think we will like that on purpose, so should be fine without sorted.